index.js

```diff
@@ -85,6 +85,12 @@ Mailgen.prototype.generatePlaintext = function (params) {
     // Render the plaintext theme with ejs, injecting the data accordingly
     var output = ejs.render(this.cachedPlaintextTheme, ejsParams);
 
+    // Strip all linebreaks from the rendered plaintext
+    output = output.replace(/(?:\r\n|\r|\n)/g, '');
+    
+    // Replace htl break tags with linebreaks
+    output = output.replace(/(?:\<br\s*\/\>)/g, '\n');
```

Comment:
This looks good, however, we want to maintain backward compatibility with custom themes that don't have `<br />` in their syntax to signal a line break. So let's simply wrap the logic you wrote on line 89 and 92 with an `if` that checks whether `this.cachedPlaintextTheme` contains at least one `<br />`.